mne/viz/ica.py

```diff
@@ -617,6 +617,8 @@ def plot_ica_overlay(ica, inst, exclude=None, picks=None, start=None,
         picks = [inst.ch_names.index(k) for k in ica.ch_names]
     if exclude is None:
         exclude = ica.exclude
+    if isinstance(exclude, int):
```

Comment:
this is not the right way to check for an int. 

Also I'd rather have an exception raised if you don't pass an array-like of int | None

Reply:
What's the right way? Looking at the [rest of the codebase](https://github.com/mne-tools/mne-python/blob/197a8418672a8e69c65a1940e45da4966decf45f/mne/stats/cluster_level.py#L186), it seems this is how it's done ...

sure, we can raise an exception